### DISCUSSION TITLE
Droppable Gear: Single Target damage + self protection

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#351] [Ideas]
Upgrades: Sharpened, Reinforced

**Comment by doubledh** [comment]
trench shovel.
dig your fortifications but also debris -> enemy :V

**Reply by ntseng** [reply]
I ended up designing a "Spiked Shield" in the gear overhaul that fulfills this profile. It grants the user protection, then hits the targeted foe for damage equal to the user's protection.
